xocto/fields/postgres/ranges.py

```diff
@@ -93,7 +93,20 @@ def _upper_to_inclusive(self, value: pg_ranges.DateRange) -> datetime.date:
         return value.upper - datetime.timedelta(days=1)
 
 
-class FiniteDateTimeRangeField(pg_fields.DateTimeRangeField):
+class _ConvertibleDateTimeRangeMixin:
+    def __init__(self, *args: Any, as_localtime: bool = True, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+        self._as_localtime = as_localtime
```

Comment:
**suggestion**: rather than a boolean, should we consider having a timezone argument? The "default" can be the localtime timezone - but you can also pass in UTC.

That way we can progressively update callers to using UTC?

That said it's probably unlikely callers would care about anything other than LOCAL and UTC right? So that might not be better..

Reply:
Yes I think it's a great idea! This allows more flexibility if the user wants.

> That said it's probably unlikely callers would care about anything other than LOCAL and UTC right?

I agree, but with hindsight it seems a bit odd to me adding a boolean only for UTC/LOCAL conversion as `as_localtime` allows to choose a timezone.